assets/app/crowbar-app.module.js

```diff
@@ -36,4 +37,27 @@
             $urlRouterProvider.otherwise('/upgrade/landing');
         }
     }
+
+    run.$inject = ['$rootScope', '$state', 'upgradeFactory', 'upgradeStepsFactory'];
+
+    function run($rootScope, $state, upgradeFactory, upgradeStepsFactory) {
```

Comment:
Also, how could this be tested?
According to John Papa, a factory could be used to solve the problem: https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#run-blocks